### PR TITLE
Fix crash in RowContainer::storeWithNulls<TypeKind::HUGEINT>

### PR DESCRIPTION
**Pull request by sjyango**
Before:
When executing the `storeWithNulls` function on dictionary input, The `DecodedVector::valueAt` function will execute `index(idx)` to retrieve the corresponding index from `indices_`. However, if no null check is performed, the index is not guaranteed to be good, and performing offset calculation will result in a core dump.

After:
Check null values first, zero out the value if it is null.

### CODE REVIEW

**Comment by sjyango** [comment]
@Yuhta I have developed a unit test to reproduce the scenario of this issue. Before this PR, a core dump will be performed, and it will run normally after this PR. Could you please review my code? Thanks.

**Comment by sjyango** [comment]
@mbasmanova Hello, there is another core dump bug here. Can you please review my codes when you have time? Thanks.

**Review by Yuhta — APPROVED** [review]

File: velox/exec/tests/RowContainerTest.cpp [hunk 1]
```diff
@@ -1968,3 +1968,51 @@ TEST_F(RowContainerTest, nextRowVector) {
...
+  }
+  SelectivityVector allRows(kNumRows);
+  DecodedVector decoded(*source, allRows);
+  for (auto i = 0; i < kNumRows; ++i) {
+    data->store(decoded, i, rows[i], kColumnIndex);
+  }
+  auto extracted = BaseVector::copy(*source);
+  data->extractColumn(rows.data(), kNumRows, kColumnIndex, extracted);
+  assertEqualVectors(source, extracted);
+}
```
[the remark this answers is not in the document]
Comment (sjyango) [reply velox/exec/tests/RowContainerTest.cpp]: done

**Comment by sjyango** [comment]
@Yuhta I noticed that the modification has been merged into the main branch. Do you need to close the pr?

**Comment by Yuhta** [comment]
@sjyango Yes there is some glitch in the tooling, we can close the PR manually

**Pull request closed without merging**